moped-editor/src/views/staff/StaffForm.js

```diff
@@ -97,6 +97,7 @@ const StaffForm = ({
       ...initialFormValues,
       // Roles are stored as an array in the DB but we need to feed the form a string
       roles: findHighestRole(initialFormValues.roles),
+      workgroup_id: initialFormValues?.moped_workgroup?.workgroup_id ?? null,
```

Comment:
not sure if this error predates your PR, but i'm seeing MUI errors that i think are related to this. changing `null` to `""` seems to fix it

> `value` prop on `input` should not be null.

and 

>  You have provided an out-of-range value `null` for the select component.